pynapple/core/interval_set.py

```diff
@@ -533,7 +550,7 @@ def drop_short_intervals(self, threshold, time_units="s"):
         threshold = TsIndex.format_timestamps(
             np.array([threshold], dtype=np.float64), time_units
         )[0]
-        return self[(self.values[:, 1] - self.values[:, 0]) > threshold]
+        return self[(self.values[:, 1] - self.values[:, 0]) > threshold, :]
```

Comment:
Why is it needed?

Reply:
this might be an artifact to before I changed the indexing, where the second slice was to grab the metadata columns. I don't think it's necessary now

Reply:
I took out this slice (and the slice in drop_long_intervals)